Delete.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Heh, they're capitalized for consistency with *our* other projects 😄 
No problem, I can change them on this branch.